group-coordinator/src/main/resources/common/message/ConsumerGroupPartitionMetadataValue.json

```diff
@@ -27,14 +27,7 @@
       { "name": "TopicName", "versions": "0+", "type": "string",
         "about": "The topic name." },
       { "name": "NumPartitions", "versions": "0+", "type": "int32",
-        "about": "The number of partitions of the topic." },
```

Comment:
Same. Unfortunately, we cannot remove fields because we released it. We must keep it. Let’s put a comment explaining that we don’t use it.